composer.json

```diff
@@ -5,14 +5,14 @@
   "homepage": "https://github.com/google/web-stories-wp",
   "license": "Apache-2.0",
   "require": {
-    "php": "^5.6 || ^7.0 || ^8.0",
+    "php": "^5.6.20 || ^7.0 || ^8.0",
```

Comment:
Why was this needed? Or did you just bump this while you were at it?

Reply:
I was seeing terminal. 
```
Loading composer repositories with package information
Updating dependencies
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Root composer.json requires sabberworm/php-css-parser dev-master#bfdd976 -> satisfiable by sabberworm/php-css-parser[dev-master].
    - sabberworm/php-css-parser dev-master requires php >=5.6.20 -> your php version (5.6; overridden via config.platform, actual: 7.4.15) does not satisfy that requirement.
  Problem 2
    - sabberworm/php-css-parser dev-master requires php >=5.6.20 -> your php version (5.6; overridden via config.platform, actual: 7.4.15) does not satisfy that requirement.
    - ampproject/amp-wp dev-develop requires sabberworm/php-css-parser dev-master#bfdd976 -> satisfiable by sabberworm/php-css-parser[dev-master].
    - Root composer.json requires ampproject/amp-wp dev-develop -> satisfiable by ampproject/amp-wp[dev-develop].
```

Reply:
Also worth noting that 5.6.20, is min required for WordPress. 
https://github.com/WordPress/wordpress-develop/blob/aa62fdbfaad20eadf10ddad11d4be0279e6cb740/src/wp-includes/version.php#L37

Reply:
Ah, looks like a recent addition: https://github.com/sabberworm/PHP-CSS-Parser/pull/215

All good then 👍